aws-spot-instances.md

```diff
@@ -23,6 +31,69 @@ Spot data feeds are an account level setting, not a payer level. Every AWS Accou
 For Spot data written to an S3 bucket only accessed by Kubecost, it is safe to delete objects after three days of retention.
 {% endhint %}
 
+## Configuring IAM
+
+Kubecost requires read access to the Spot data feed bucket. The following IAM policy can be used to grant Kubecost read access to the Spot data feed bucket.
+
+```json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+        "Sid": "SpotDataFeed",
+        "Effect": "Allow",
+        "Action": [
```

Comment:
Suggest revising permissions to the following:

```
"s3:GetBucketAcl",
"s3:GetBucketLocation",
"s3:GetObject",
"s3:ListBucket"
```

Reply:
@srpomeroy I used `policy/kc-spot-feed-us-east-2-policy` as reference for this doc, and I also used that in my repro. Have you tested the above permissions? If so I can make the change, as your suggestion would be more restrictive!